extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/discovery/AstroDiscoveryService.java

```diff
@@ -62,10 +62,10 @@ protected void startScan() {
     @Override
     protected void startBackgroundDiscovery() {
         if (astroDiscoveryJob == null) {
-            astroDiscoveryJob = scheduler.scheduleAtFixedRate(() -> {
+            astroDiscoveryJob = scheduler.scheduleWithFixedDelay(() -> {
```

Comment:
> I am not sure which one would be appropriate for this scenario.

But I am sure, just check [D.2 of our guidelines](https://www.eclipse.org/smarthome/documentation/development/guidelines.html#d-runtime-behavior) 😎